Test/Mono.Cecil.Tests.csproj

```diff
@@ -7,6 +7,10 @@
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp">
       <Version>2.10.0</Version>
     </PackageReference>
+    <PackageReference Include="runtime.$(NetCoreSdkPortableRuntimeIdentifier).Microsoft.NETCore.ILAsm">
```

Comment:
It's not common to directly reference a runtime.* package. Is that necessary here?

Reply:
yeah. the main package uses runtime.json so it doesn't work without a publish: https://github.com/dotnet/sdk/issues/35792